corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -1,5 +1,5 @@
 /* global Uint8Array */
-hqDefine("app_manager/js/details/case_claim", function () {
+hqDefine("app_manager/js/details/case_claim",['hqwebapp/js/assert_properties'], function ( assertProperties ) {
```

Comment:
This is normally the right thing to do, but app manager isn't migrated to RequireJS yet, so it's better to remove it here and instead use `hqImport` to bring it in. There are some notes on this here: https://github.com/dimagi/commcare-hq/blob/master/docs/js-guide/migrating.md#note-on-editing-unmigrated-javascript